bluetooth.py: migrate to ravel

thoradia opened the pull request:
I submit this pull request to indicate that I am working on bluetooth settings
The changes do not break the settings add-on
They unfortunately do not fix it either
Work in progress, thus

thoradia commented:
This re-enables basic functions such as connecting/pairing/disconnecting
Please test and merge

chewitt commented on resources/lib/dbus_bluez.py:
```diff
@@ -0,0 +1,99 @@
+import dbus_utils
```
copyright header

chewitt commented on resources/lib/modules/bluetooth.py:
```diff
@@ -73,48 +76,26 @@ def exit(self):
-        oe.dbg_log('bluetooth::adapter_info', 'exit_function', oe.LOGDEBUG)
-        return res
+    def init_adapter(self):
+        dbus_bluez.adapter_set_alias(self.dbusBluezAdapter, 'LibreELEC')
```
hardcoded LibreELEC should be replaced with @@DISTRONAME@@ or something we can sed at build time to support downstream..

thoradia commented:
Thank you for the feedback @chewitt 
Bluetooth now advertises the hostname or the distribution name
Headers have been added
I also refactored hostname/regdomain logic and fixed connman error handling
This can be merged
